### ISSUE TITLE
Impossible to set help message in nested form

### ISSUE THREAD

**Issue by mikemix** [sonata-project/SonataAdminBundle#5648]
### Environment

Docker `php:7.3-apache` image

#### Sonata packages

```
$ composer show --latest 'sonata-project/*'
sonata-project/admin-bundle              3.51.0 3.51.0 The missing Symfony Admin Generator
sonata-project/block-bundle              3.15.0 3.15.0 Symfony SonataBlockBundle
sonata-project/cache                     2.0.1  2.0.1  Cache library
sonata-project/core-bundle               3.17.0 3.17.0 Symfony SonataCoreBundle
sonata-project/datagrid-bundle           2.5.0  3.0.0  Symfony SonataDatagridBundle
sonata-project/doctrine-extensions       1.3.0  1.3.0  Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 3.10.0 3.10.0 Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/exporter                  2.0.1  2.0.1  Lightweight Exporter library
```

#### Symfony packages

```
$ composer show --latest 'symfony/*'
symfony/asset                 v4.2.10 v4.2.10 Symfony Asset Component
symfony/browser-kit           v4.2.10 v4.2.10 Symfony BrowserKit Component
symfony/cache                 v4.2.10 v4.2.10 Symfony Cache component with PSR-6, PSR-16, and tags
symfony/class-loader          v3.4.29 v3.4.29 Symfony ClassLoader Component
symfony/config                v4.2.10 v4.2.10 Symfony Config Component
symfony/console               v4.2.10 v4.2.10 Symfony Console Component
symfony/contracts             v1.1.5  v1.1.5  A set of abstractions extracted out of the Symfony components
symfony/css-selector          v4.2.10 v4.2.10 Symfony CssSelector Component
symfony/debug                 v4.2.10 v4.2.10 Symfony Debug Component
symfony/dependency-injection  v4.2.10 v4.2.10 Symfony DependencyInjection Component
symfony/doctrine-bridge       v4.2.10 v4.2.10 Symfony Doctrine Bridge
symfony/dom-crawler           v4.2.10 v4.2.10 Symfony DomCrawler Component
symfony/dotenv                v4.2.10 v4.2.10 Registers environment variables from a .env file
symfony/event-dispatcher      v4.2.10 v4.2.10 Symfony EventDispatcher Component
symfony/expression-language   v4.2.10 v4.2.10 Symfony ExpressionLanguage Component
symfony/filesystem            v4.2.10 v4.2.10 Symfony Filesystem Component
symfony/finder                v4.2.10 v4.2.10 Symfony Finder Component
symfony/flex                  v1.4.1  v1.4.5  Composer plugin for Symfony
symfony/form                  v4.2.10 v4.2.10 Symfony Form Component
symfony/framework-bundle      v4.2.10 v4.2.10 Symfony FrameworkBundle
symfony/http-foundation       v4.2.10 v4.2.10 Symfony HttpFoundation Component
symfony/http-kernel           v4.2.10 v4.2.10 Symfony HttpKernel Component
symfony/inflector             v4.2.10 v4.2.10 Symfony Inflector Component
symfony/intl                  v4.2.10 v4.2.10 A PHP replacement layer for the C intl extension that includes additional data from the ICU library.
symfony/messenger             v4.2.10 v4.2.10 Symfony Messenger Component
symfony/monolog-bridge        v4.2.10 v4.2.10 Symfony Monolog Bridge
symfony/monolog-bundle        v3.4.0  v3.4.0  Symfony MonologBundle
symfony/options-resolver      v4.2.10 v4.2.10 Symfony OptionsResolver Component
symfony/orm-pack              v1.0.6  v1.0.6  A pack for the Doctrine ORM
symfony/phpunit-bridge        v4.3.2  v4.3.2  Symfony PHPUnit Bridge
symfony/polyfill-intl-icu     v1.11.0 v1.11.0 Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-intl-idn     v1.11.0 v1.11.0 Symfony polyfill for intl's idn_to_ascii and idn_to_utf8 functions
symfony/polyfill-mbstring     v1.11.0 v1.11.0 Symfony polyfill for the Mbstring extension
symfony/polyfill-php72        v1.11.0 v1.11.0 Symfony polyfill backporting some PHP 7.2+ features to lower PHP versions
symfony/process               v4.2.10 v4.2.10 Symfony Process Component
symfony/profiler-pack         v1.0.4  v1.0.4  A pack for the Symfony web profiler
symfony/property-access       v4.2.10 v4.2.10 Symfony PropertyAccess Component
symfony/property-info         v4.2.10 v4.2.10 Symfony Property Info Component
symfony/proxy-manager-bridge  v4.2.10 v4.2.10 Symfony ProxyManager Bridge
symfony/routing               v4.2.10 v4.2.10 Symfony Routing Component
symfony/security-acl          v3.0.2  v3.0.2  Symfony Security Component - ACL (Access Control List)
symfony/security-bundle       v4.2.10 v4.2.10 Symfony SecurityBundle
symfony/security-core         v4.2.10 v4.2.10 Symfony Security Component - Core Library
symfony/security-csrf         v4.2.10 v4.2.10 Symfony Security Component - CSRF Library
symfony/security-guard        v4.2.10 v4.2.10 Symfony Security Component - Guard
symfony/security-http         v4.2.10 v4.2.10 Symfony Security Component - HTTP Integration
symfony/serializer            v4.2.10 v4.2.10 Symfony Serializer Component
symfony/serializer-pack       v1.0.2  v1.0.2  A pack for the Symfony serializer
symfony/stopwatch             v4.2.10 v4.2.10 Symfony Stopwatch Component
symfony/swiftmailer-bundle    v3.2.8  v3.2.8  Symfony SwiftmailerBundle
symfony/templating            v4.2.10 v4.2.10 Symfony Templating Component
symfony/test-pack             v1.0.6  v1.0.6  A pack for functional and end-to-end testing within a Symfony app
symfony/translation           v4.2.10 v4.2.10 Symfony Translation Component
symfony/twig-bridge           v4.2.10 v4.2.10 Symfony Twig Bridge
symfony/twig-bundle           v4.2.10 v4.2.10 Symfony TwigBundle
symfony/validator             v4.2.10 v4.2.10 Symfony Validator Component
symfony/var-dumper            v4.2.10 v4.2.10 Symfony mechanism for exploring and dumping PHP variables
symfony/var-exporter          v4.2.10 v4.2.10 A blend of var_export() + serialize() to turn any serializable data structure to plain PHP code
symfony/web-link              v4.2.10 v4.2.10 Symfony WebLink Component
symfony/web-profiler-bundle   v4.2.10 v4.2.10 Symfony WebProfilerBundle
symfony/webpack-encore-bundle v1.6.2  v1.6.2  Integration with your Symfony app & Webpack Encore!
symfony/yaml                  v4.2.10 v4.2.10 Symfony Yaml Component
```

#### PHP version

```
$ php -v
PHP 7.3.5 (cli) (built: May  8 2019 02:40:53) ( NTS )
```

## Subject

Sonata ignores help messages in sub forms.

## Steps to reproduce

I have a simple form type which I use extensively in admin classes:

```php
final class MetaFormType extends AbstractType
{
    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder
            ->add('title', TextType::class, [
                'required' => false,
                'constraints' => new Length(['max' => 255]),
                'help' => 'help',
            ])
            ->add('description', TextType::class, [
                'required' => false,
                'constraints' => new Length(['max' => 255]),
                'help' => 'help',
            ])
            ->add('keywords', TextType::class, [
                'required' => false,
                'constraints' => new Length(['max' => 255]),
                'help' => 'help',
            ]);
    }

    public function configureOptions(OptionsResolver $resolver)
    {
        $resolver->setDefault('data_class', Meta::class);
    }

    public function getParent()
    {
        return FormType::class;
    }
}
```

I expect Sonata to render the help message, but none are visible. Nor can I set the help message using any method from the documentation https://symfony.com/doc/master/bundles/SonataAdminBundle/reference/form_help_message.html

## Expected results

Sonata should render default help messages and allow to override from the admin.

**Comment by core23** [comment]
You're mixing to different things. The docs you are refering is about the `FormMapper`, not the form types

**Comment by mikemix** [comment]
But still, shouldn't Sonata display the help message for each field?

**Comment by GeneraleCauchemar** [comment]
Hi @mikemix ,

**Sonata**: 3.51.0
**PHP**: 7.2.20
**Symfony**: 4.3.3

I've had the exact same problem here. After fiddling with it for a bit, I've found a workaround.
In your form type, add a `sonata_help` option with your help message as value.

Here is my abstract Sonata admin :
```php
abstract class AbstractPageAdmin extends AbstractAdmin
{
    protected function configureFormFields(FormMapper $formMapper)
    {
        $builderFactory = $formMapper->getFormBuilder()->getFormFactory();
        $pageBuilder     = $builderFactory->createBuilder(PageType::class);

        $formMapper
            ->with('admin.group.title')
            ->add($pageBuilder->get('title'))
            ->end()
}
```

And here is the FormType I use to define the title and some other fields:
```php
class PageType extends AbstractType
{
    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder
            ->add('title', TextType::class, [
                'attr'               => ['class' => 'input-lg'],
                'label'              => false,
                'sonata_help'   => 'Help message',
                'required'         => true,
            ])
        ;
    }
}
```

The only problem is that the `sonata_help` field cannot be translated unless you override the template rendering it.

**TL;DR: just adding the `sonata_help` option in my form type did the trick, even though it's not the cleanest way to do it.**
